internal/pkg/mount/v2/mount.go

```diff
@@ -290,14 +299,23 @@ func (p *Point) Move(newTarget string) error {
 }
 
 func (p *Point) mount() error {
-	return unix.Mount(p.source, p.target, p.fstype, p.flags, p.data)
+	if err := unix.Mount(p.source, p.target, p.fstype, p.flags, p.data); err != nil {
+		return err
+	}
+
+	if p.selinuxLabel != "" {
+		return selinux.SetLabel(p.target, p.selinuxLabel)
+	}
+
+	return nil
 }
 
 func (p *Point) unmount(printer func(string, ...any)) error {
 	return SafeUnmount(context.Background(), printer, p.target)
 }
 
 func (p *Point) share() error {
+	// TODO: SELinux label?
```

Comment:
it only marks as shared, so it happens only on top of regular `mount()` operation

Reply:
Thanks, will remove the comment